### DISCUSSION TITLE
HowTo set up Mac to be able to run python -m scripts.score_real_collections?

### DISCUSSION THREAD

**Discussion by bmm-brobbins** [OpenRarity/open-rarity#104] [Q&A] [answered]
I wonder if some kind soul could publish a list of steps to get a mac set up to run openRarity's scripts, e.g. `python -m scripts.score_real_collections`

With the entire installation preferably residing in some kind of self-contained "virtual environment"?

Thank you

**Comment by bmm-brobbins** [comment]
Anybody?

All I ever get when running `python -m scripts.score_real_collections` is `Error while finding module specification for 'scripts.score_real_collections' (ModuleNotFoundError: No module named 'scripts')`

I have open-rarity installed via pip.  I have Python 3.11.0 (v3.11.0:deaf509e8f, Oct 24 2022, 14:43:23) [Clang 13.0.0 (clang-1300.0.29.30)] on darwin

Need I also do something with poetry?  If so, what?

Just a list of steps starting from scratch on MacOS please.

> I wonder if some kind soul could publish a list of steps to get a mac set up to run openRarity's scripts, e.g. `python -m scripts.score_real_collections`
> 
> With the entire installation preferably residing in some kind of self-contained "virtual environment"?

**Comment by impreso** [comment]
I can reproduce this issue - thanks for reporting! Will follow up with how to fix it on mac.

**Comment by impreso** [comment]
Could you pls clone the OpenRarity repo and use the mentioned command from the cloned repository?

**Reply by bmm-brobbins** [reply]
> Could you pls clone the OpenRarity repo and use the mentioned command from the cloned repository?

Yes!  Got it to work with the cloned repo technique.  Thank you very much!

**Comment by impreso** [comment, marked as answer]
We are working on the next release of the library and will address it with the proper CLI implementation.
